packages/form-client/style/form-view.css

```diff
@@ -6,32 +6,6 @@
     box-sizing: border-box;
 }
 
-.form-editor-general {
-    display: table;
-    width: 100%;
-}
-
-.form-editor-general > div {
-    display: table-row;
-}
-
-.form-editor-general label {
-    display: table-cell;
-    padding-right: 10px;
-    vertical-align: top;
-    width: 50px;
-    margin-bottom: 5px;
-}
-
-.form-editor-general input,
-.form-editor-general textarea {
-    display: table-cell;
-    width: 80%;
-    border: gray 1px solid;
-    resize: none;
-    margin-bottom: 5px;
-}
-
```

Comment:
Is there a particular reason you left `.form-editor-attributes-tab` and `.react-tabs__tab > h3` in this file? I believe `.form-editor-attributes-tab` could probably be removed as it is not used anywhere (anymore) and the other one should probably also go into the React package.

In general, we should think about styling a bit more as currently the UI only looks good in 'light' theme but switching to dark theme some parts become unreadable. This should be done in a separate issue, of course.

Reply:
No particular reason, I just moved the css files. I will clean it up, thanks!

Reply:
In the last checkin the css files are moved and removed where needed.